src/bundle/Resources/views/admin/search/search_form.html.twig

```diff
@@ -0,0 +1,14 @@
+{{ form_start(form) }}
+<div class="input-group">
+    {{ form_widget(form.query, {'attr': {'class': 'form-control'} }) }}
+    {{ form_widget(form.page, {'attr': {'value': 1 }}) }}
```

Comment:
We shouldn't overwrite the value in the view. Value can be passed in the controller by changing corresponding data object.